cms/controllers/Render.php

```diff
@@ -133,10 +133,15 @@ public function page()
 
         // --------------------------------------------------------------------------
 
-        $this->oMetaData
-            ->setTitles([$oData->seo_title ?: $oData->title])
-            ->setDescription($oData->seo_description)
-            ->setKeywords(explode(',', $oData->seo_keywords));
+        $this->oMetaData->setTitles([$oData->seo_title ?: $oData->title]);
+
+        if ($oData->seo_description) {
+            $this->oMetaData->setDescription($oData->seo_description ?: $oData->description);
+        }
+
+        if ($oData->seo_keywords) {
+            $this->oMetaData->setKeywords(explode(',', $oData->seo_keywords ?: $oData->keywords));
```

Comment:
1. inline ternary, don't think the fallbacks exist
2. instead of explode I suggest using `preg_split` to split on multiple potential delimiters a user might use:
```
preg_split('/[,; ]/', $oData->seo_keywords)
```